cli/src/etos_client/downloader/downloader.py

```diff
@@ -142,6 +107,23 @@ def __save_file(self, item: Downloadable, response: requests.Response) -> None:
         with open(download_path, "wb+") as report:
             for chunk in response:
                 report.write(chunk)
+        if not self.__verify(item, download_path):
```

Comment:
We should probably allow the checksum to be empty and not be verified, for the cases where ETOS is not the producer of the file.